File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        my_dict = {}
        for i in range(0, len(nums)):
            if my_dict.get(nums[i], 0) > 0:
                return True
            my_dict[nums[i]] = my_dict.get(nums[i], 0) + 1
        return False

        

File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {

        sort(nums.begin(), nums.end());//[0,1,2,3] = 4
        for(int i = 0 ; i < nums.size()-1 ; ++i){
            if(nums[i] == nums[i+1]) return true;
        }
        return false;
    }
};